src/openrct2/network/network.cpp

```diff
@@ -2134,7 +2134,11 @@ sint32 network_begin_client(const char *host, sint32 port)
 
 sint32 network_begin_server(sint32 port)
 {
-	return gNetwork.BeginServer(port);
+	char *address = nullptr;
+	if (strlen(gConfigNetwork.listen_address) > 0)
+		address = gConfigNetwork.listen_address;
+
+	return gNetwork.BeginServer(port, address);
```

Comment:
Okay, didn't dug that much into the source code. Will do that later, I'm busy again.